VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -201,6 +201,40 @@ describe('ParticipantsPanelComponent', () => {
         expect(videocallService.muteParticipant).toHaveBeenCalledWith(pat.pexipId, false);
     });
 
+    it('should not mute conference when any of the second last participant is unmuted manually', () => {
+        videocallService.muteAllParticipants.calls.reset();
+        component.isMuteAll = true;
+        // Mute all participants except one
+        for (let index = 0; index < component.participants.length - 1; index++) {
+            component.participants[index].isMuted = true;
+            (<ParticipantPanelModel>component.participants[index]).status = ParticipantStatus.InHearing;
+        }
+
+        // Get any muted participant
+        const pat = component.participants[0];
+        // Unmute the participant
+        component.toggleMuteParticipant(pat);
+
+        expect(videocallService.muteAllParticipants).toHaveBeenCalledTimes(0);
+    });
+
+    it('should not mute conference when any of the second last participant is muted manually', () => {
+        videocallService.muteAllParticipants.calls.reset();
+        component.isMuteAll = true;
+        // Mute all participants except one
```

Comment:
this comment does not reflect the setup here.

Reply:
Updated.